src/signer/src/intTest/java/org/niis/xroad/signer/test/glue/SignerStepDefs.java

```diff
@@ -273,7 +273,7 @@ public void selfSignedCertGeneratedForTokenKeyForClient(String friendlyName, Str
         final KeyInfo keyInToken = findKeyInToken(friendlyName, keyName);
 
         scenarioCert = SignerProxy.generateSelfSignedCert(keyInToken.getId(), getClientId(client), KeyUsageInfo.SIGNING,
-                "CN=" + client, Date.from(now().minus(5, DAYS)), Date.from(now().plus(5, DAYS)));
+                client, Date.from(now().minus(5, DAYS)), Date.from(now().plus(5, DAYS)));
```

Comment:
why this was required?

Reply:
because later one same prefix is added by other method again and X509Name constructor is unhappy when it get CN=CN=something